entities/markets/userBets.ts

```diff
@@ -6,3 +8,32 @@ export interface UserBets extends UserPosition {
   fpmm: FixedProductMarketMaker;
   condition: Condition;
 }
+
+export class Bets {
```

Comment:
File is named UserBets but the class is named Bets, should we call it UserBets too? I think this is related to a user positions, so UserBets makes sense in my mind.

Reply:
Yeah agree, and if in the future we add another feature like a ranking or sharing bets, or something like that, we can differentiate one Bet from another with the naming suggested by Leo.

Reply:
Yeah, makes sense, I will update it to `UserBets`.